src/index.js

```diff
@@ -13,9 +13,10 @@ function isReactFragment(openingElement) {
 function applyAttribute({ openingElement, t, name, options }) {
   if (!openingElement || isReactFragment(openingElement)) return
 
-  const isAttributeAlreadySet = openingElement.node.attributes.find(
-    node => node.name.name === options.attribute
-  )
+  const isAttributeAlreadySet = openingElement.node.attributes.find(node => {
+    if (!node.name) return // fix build issues for wrapped components
```

Comment:
Happy to add the condition, could you remove the comment or make it more generic?

Reply:
Sure thing, just added for context for you :)

Reply:
Done :)